lib/utils/decode.js

```diff
@@ -1,4 +1,8 @@
-const { TextDecoder } = require('util')
+let TextDecoder
```

Comment:
so when do we set this if it's not available?

Reply:
see below